pom.xml

```diff
@@ -87,7 +87,9 @@
                 <version>${tycho-version}</version>
                 <configuration>
                     <target>
-                        <file>${basedir}/../net.sf.eclipsecs.target/net.sf.eclipsecs.target.target</file>
+                        <!-- TODO replace multi module dir by basedir after Tycho 3.0+
+                        https://github.com/eclipse-tycho/tycho/issues/1660 -->
```

Comment:
I know that most Checks disabled.
Todo is not allowed in final code.
Can you make it ` until https://github.com/eclipse-tycho/tycho/issues/1660` ?

Reply:
I've reworded it, without TODO. "until URL" would be a bit confusing, since I have already fixed that bug in Tycho, but we will not be able to use the bugfix version anytime soon (remember the Java 17 dependency, when I tried to upgrade recently).

Reply:
In this case we should create issue in our repo to migrate to jdk 17, and make comment to describe until it.